source/community/release_notes/ulmo/ulmo_rp.rst

```diff
@@ -1,6 +1,137 @@
+shared roles & permissions service and connecting it to Libraries, so the same
+model can be extended to other parts of the platform over time.
+
+.. image:: /_images/educator_how_tos/library_team_member_tab.png
+   :alt: The Team Members tab of the Admin Console, showing two team members in a table with the columns Name, Email, Role, and Actions
+   :width: 800
+
+Any user who has a role in a library can open the library team manager in the
+new Administrative Console from Studio. This view shows all members of the
+library, the role each person holds, and the permissions associated with each
+role. Library Admins and global site admins can use this view to assign or revoke
+roles for that library, so team membership and role information live in a single
+place.
+
+The goal of the Ulmo MVP is to introduce the new roles and permissions model to
+Libraries with functional parity and a clearer model of roles, not to change how
+authors create or reuse content.
+
+.. admonition:: Migrate Legacy Libraries
+
+    :ref:`Legacy Libraries <Legacy Content Libraries Overview>` do not support the new Roles & Permissions functionality.
+    Upgrade Legacy Libraries in order to take advantage of this new feature!
+    To learn more view the :ref:`migration documentation <Migrating Legacy Libraries>`.
+
+What's Available in Ulmo
+************************
+
+The Ulmo release includes:
+
+* :ref:`Library scoped roles <Library Roles>`: Library Admin, Library Author, Library Contributor
+  and Library User, each mapped to a defined set of permissions that control who
+  can view, edit, publish or reuse library content, and who can manage the
+  library team.
+
+* A new :ref:`Library Contributor` role that matches most of the Library Author
+  capabilities for creating and editing content, managing tags and collections,
+  and reusing content, but cannot publish content.
+
+* Enforcement of these permissions when users view a library, edit draft
+  content, publish content, manage tags and collections, reuse library content
+  in courses, or manage the library team.
+
+* A library team manager in the :ref:`Administrative Console <Ulmo console>`.
+  Any user who has a role in a library can open this view to see all members,
+  their roles and the available role definitions. Library Admins and global
+  site admins use the same view to assign or revoke roles for that library, so team
+  membership and role information are managed from a single place.
+
+* An automatic migration tool that replicates existing library roles into the new system,
+  so current configurations are preserved without manual changes.
+
+Scope and Impact
+****************
+
+The new roles and permissions system applies only to Content Libraries, not
+:ref:`Legacy Libraries <Legacy Content Libraries Overview>`. It replaces the
+previous library specific authorization logic with library-scoped roles managed
+by the shared authorization service and surfaced through the :ref:`Administrative
```

Comment:
What do we mean by "the shared authorization service"? It isn't clear to me :S

Are we referring to whether we support the old and new authorization systems? If so, maybe we can phrase it better.

What do you think @gviedma-aulasneo?